R/fotmob_leagues.R

```diff
@@ -153,15 +153,26 @@ fotmob_get_league_ids <- function(cached = TRUE) {
 }
 
 #' @importFrom purrr safely
-.fotmob_get_league_resp <- function(league_id, page_url, fallback = TRUE) {
-  url <- sprintf("https://www.fotmob.com/api/leagues?id=%s", league_id)
+#' @importFrom httr parse_url build_url
+.fotmob_get_league_resp <- function(league_id, page_url, season = NULL, fallback = TRUE) {
+  url <- httr::parse_url("https://www.fotmob.com/api/leagues")
+  url$query <- list(
+    "id" = league_id,
+    "season" = season
+  )
+  url <- httr::build_url(url)
```

Comment:
building the URL like this is advantageous since we may not have a value for `season`. `build_url()` knows to ignore a `NULL`